src/response_code.rs

```diff
@@ -49,10 +49,11 @@ pub enum Tss2ResponseCode {
     Success,
     FormatZero(FormatZeroResponseCode),
     FormatOne(FormatOneResponseCode),
+    WrapperError(Tss2ResponseCodeKind),
```

Comment:
As this enum list all of the TSS 2.0 specs defined response codes, would it maybe be better to contain a `Tss2ResponseCode` variant inside a bigger enumeration?
It could even be in another file, named `error.rs` which contains an `Error` enum:
```rust
pub enum Error {
    LocalError(LocalErrorKind),
    Tss2Error(Tss2ResponseCode),
};
```
so that our functions (or only the most high-level ones) could return that?

I am just afraid that we are mixing something that is defined by a spec and our own error code in the same module.

Reply:
Done!